src/atoms/Layout/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from './layout.module.scss';
+
+function Layout ( props ) {
+  const {
+    children,
+    childCols,
+    breakpoint,
+    horizontalAlign,
+    verticalAlign
+  } = props;
+
+  let colsIdx = -1;
+
+  const kids = React.Children.map( children, ( child, idx ) => {
+    const length = !children.length ? 1 : children.length;
+    const childColsLength = childCols.length;
+
+    colsIdx = childColsLength == 1 || ( (idx + 1) > childColsLength && (idx + 1) % childColsLength === 1 ) ? -1 : colsIdx;
```

Comment:
`childColsLength` should be an integer at this point, so you should be able to strictly compare (`===`)